.eslintrc.json

```diff
@@ -71,13 +88,8 @@
                 "allowTemplateLiterals": true
             }
         ],
-        "no-unused-vars": [
-            "error"
-        ],
-        "brace-style": [
-            "error",
-            "stroustrup"
-        ],
+        "no-unused-vars": ["error"],
+        "brace-style": ["error", "stroustrup"],
```

Comment:
This conflicts with `prettier`. If I have
```javascript
if (...) {
    // ...
} else {
    // ...
}
```
I get the error `error    Closing curly brace appears on the same line as the subsequent block  brace-style`. If however I switch to
```javascript
if (...) {
    // ...
}
else {
    // ...
}
```
I get `warning  Insert `⏎···················`                                         prettier/prettier`.

There has to be at least one correct way. Either we switch this to `1tbs` or we change the `prettier` settings to allow for the Stroustrup style.

Reply:
Awesome, I'll change this to `1tbs`.  I personally prefer stroustrup style but I think sticking to prettier's defaults is the right call.